mm2src/mm2_main/tests/integration_tests_common/mod.rs

```diff
@@ -116,8 +116,9 @@ pub async fn enable_utxo_v2_electrum(
     coin: &str,
     servers: Vec<Json>,
     timeout: u64,
+    activation_params_extra: Option<Json>,
```

Comment:
It's better to specify that this is for private key policy instead of any parameters.

Reply:
done